AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -49,6 +49,10 @@ disk.yandex.*##.mail360-banner
 !-----------------------------------------------
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/87556
+m.hani.co.kr##.support-header
+m.hani.co.kr##.bn-promotion-support
+m.hani.co.kr##.support-content
```

Comment:
As far as I can understand, those elements are donation widgets. Move them to the Annoyances - widgets.txt - section `Donation widgets`.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/87557/commits/e2dd1a2d69e0fbd4a8541b7e62a62b2c547da8d9